pkg/azurefile/utils.go

```diff
@@ -305,3 +305,17 @@ func replaceWithMap(str string, m map[string]string) string {
 	}
 	return str
 }
+
+func getConfidentialRuntimeClasses() []string {
+	return []string{"kata-cc", "kata-cc-isolation"}
+}
+
+// check if runtimeClass is confidential
+func isConfidentialRuntimeClass(runtimeClass string) bool {
```

Comment:
pls write unit tests as more as possible

Reply:
I have added two tests. Need help with more for staging/unstaging, publish/unplish etc.